docs/Developer-Guide_Using-Vagrant.md

```diff
@@ -54,7 +54,7 @@ The following steps are all run on the *guest* Vagrant created for us.
 Once it's finally up and you're logged in, it works much like any of the other install methods (NOTE: again, these commands are run on the *guest* box).
 
 	# Let's get building!
-	cd build
+	cd armbian
```

Comment:
"cd build" is correct

Reply:
When I follow the guide and vagrant ssh into the guest the only directory inside the guest is "armbian". This is in the Vagrantfile:
`git clone https://github.com/armbian/build /home/vagrant/armbian`
But I'm just getting started with this :)

Reply:
aha. than it might be ok :). I am not using Vagrant